website/common/locales/en/groups.json

```diff
@@ -143,6 +143,7 @@
   "PMDisabledCaptionText": "You can still send messages, but no one can send them to you.",
   "block": "Block",
   "unblock": "Un-block",
+  "blockWarning": "Block - This will have no effect if the player is a mod now or becomes a mod in future.",
```

Comment:
Can mod be changed to moderator in both places? We find some users don't know what the abbreviation means.